nilearn/plotting/tests/test_img_plotting.py

```diff
@@ -77,11 +77,8 @@ def test_plot_anat():
     import pylab as pl
     pl.switch_backend('template')
     img = _generate_img()
-    # Test saving with empty plot
-    z_slicer = plot_anat(anat_img=False, display_mode='z')
-    with tempfile.TemporaryFile(suffix='.png') as fp:
-        z_slicer.savefig(fp.name)
```

Comment:
FYI: this failing test is actually failing for a real reason because of a bug in your code. Fix it and we can merge ;)